app.js

```diff
@@ -44,14 +44,14 @@ app.use(session({
 app.use(passport.initialize());
 app.use(passport.session());
 
-app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
+app.use(favicon(path.join(__dirname, 'public/images', 'favicon.ico')));
 app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use('/dist', express.static(path.join(__dirname, '/dist')));
-app.use('/public', express.static(path.join(__dirname, '/public')));
-app.use('/bower_components', express.static(path.join(__dirname, '/bower_components')));
+app.use('/public/images', express.static(path.join(__dirname, '/public/images')));
+app.use('/node_modules/bootstrap', express.static(path.join(__dirname, '/node_modules/bootstrap')));
```

Comment:
Yeah, I'm keeping it for now. The future plan is to do 'in-house' styling and get rid of bootstrap. So that won't last too much longer